frontend/src/features/permits/components/permit-list/ApplicationsInReviewRowOptions.tsx

```diff
@@ -70,13 +67,12 @@ export const ApplicationsInReviewRowOptions = ({
   } = useUpdateApplicationInQueueStatus();
 
   useEffect(() => {
-    if (isUpdateApplicationError) {
+    if (
+      isUpdateApplicationError &&
+      updateApplicationError.response?.status === 422
+    ) {
       // if the application has already been withdrawn by another user
-      if (updateApplicationError.response?.status === 422) {
-        return setIsAIRModalOpen(true);
-      }
-      // handle all other errors
-      navigate(ERROR_ROUTES.UNEXPECTED);
+      return setIsAIRModalOpen(true);
     }
   }, [isUpdateApplicationError, updateApplicationError]);
```

Comment:
Remember what I said last time about including objects/reference types inside dependency arrays? Create a variable setting its value to ```updateApplicationError.response?.status``` and pass that variable into the dependency array instead